packages/synapse-react-client/src/components/Evaluation/EvaluationEditorPage.stories.ts

```diff
@@ -15,6 +20,69 @@ export const Entity: Story = {
 }
 
 export const Evaluation: Story = {
+  parameters: {
```

Comment:
Add some mock data to make it easier to interact with this this component in Storybook.